src/Data/Traversable.js

```diff
@@ -0,0 +1,48 @@
+/* global exports */
+"use strict";
+
+// module Data.Traversable
+
+function Cont (fn) {
```

Comment:
You could use an IIFE to avoid that:

```purescript
exports.traverseArrayImpl = (function () {
  // Cont, etc goes here
  return function (apply) {
    // ...
  };
}());

The advantage is DCE will be able to remove all this stuff then - my proposed DCE functionality will only operate on `exports.foo = ...` things, and keep everything else in the FFI files.